### ISSUE TITLE
Train FCN Alexnet model

### ISSUE THREAD

**Issue by Alqazzaz** [alexgkendall/caffe-segnet#111]
Hi,
I want to train the FCN Alexnet on my dataset that contains this dimension (3,160,216) and 5 classes. 

I got this error

```
ERROR: Check failed: status == CUBLAS_STATUS_SUCCESS (11 vs. 0) CUBLAS_STATUS_MAPPING_ERROR

conv2 needs backward computation.
norm1 needs backward computation.
pool1 needs backward computation.
relu1 needs backward computation.
conv1 needs backward computation.
shift does not need backward computation.
label_label_0_split does not need backward computation.
label does not need backward computation.
data_data_0_split does not need backward computation.
data does not need backward computation.
This network produces output accuracy
This network produces output loss
Network initialization done.
Solver scaffolding done.
Finetuning from /usr/lib/python2.7/dist-packages/digits/DIGITS-master/examples/semantic-segmentation/fcn.berkeleyvision.org-master/voc-fcn-alexnet
Starting Optimization
Solving
Learning Rate Policy: step
Iteration 0, Testing net (#0)
Check failed: status == CUBLAS_STATUS_SUCCESS (11 vs. 0)  CUBLAS_STATUS_MAPPING_ERROR
        
I appreciate any help.

**Comment by TimoSaemann** [comment]
Hi
In most cases the number of outputs doesn't match with the number of labels.

**Comment by Alqazzaz** [comment]
Thanks for your answer.
Can you give me more clarifications please??

**Comment by shivamsardana** [comment]
@TimoSaemann  I am training SegNet basic model in which I only want 4 classes not 11. How should I proceed? Please help
